Fix base64 strings

cossssmin opened the pull request:
Currently, the plugin returns an incomplete CSS value for strings containing a `;` character, like those used in base64 data.

\- Expected
\+ Received

``` diff
  Object {
    "style": Object {
-     "background": "url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkAAIAAAoAAv/lxKUAAAAASUVORK5CYII=)",
+     "background": "url(data:image/png",
    },
  }
```

cossssmin commented:
@maltsev @Scrum can you guys please review the fix? Used a negative lookahead for the delimiter regex, but maybe you have a better solution (?)

maltsev reviewed (changes requested):
Looks correct to me! Thanks for fixing it!

Does it make sense to add one more test to cover a case when there are several CSS attributes (e.g., `background: url(...); color: red;`)?

cossssmin commented:
I can update the assertion to be like the other one, sure 👍

cossssmin commented:
Updated.

maltsev commented:
Awesome!

Maybe let's wait for @Scrum review before merging this PR.

Scrum reviewed (approved):
It looks good, thanks

cossssmin commented:
Thanks for reviewing 👍

@maltsev for the backport I'll just create a `v0.x` branch with what we have today but CJS code, and and run `np` from that following [their guide](https://www.npmjs.com/package/np#release-an-update-to-an-old-major-version).

cossssmin commented:
@maltsev I've created [v0.x](https://github.com/posthtml/posthtml-attrs-parser/tree/v0.x) from which we can do patch releases for the old version.

I've used the current ESM version as a base and added `unbuild` to create ESM and CJS builds so as to not break for existing users. 

Also updated the GitHub workflow in that branch to run on any push to `v0.x` so we can run CI any time we need to backport some feature. So now basically the old version is up to date, including dependencies.

In fact, should we just merge it into `master` and publish `v1.1` for everyone instead, now that it has both ESM and CJS?

Let me know and I'll take care of everything 👍

cossssmin commented:
As a precaution, we could also publish `v0.1.2`, check that everything is working fine, then merge it into master and publish the ESM/CJS hybrid as `v1.1.0`.

Scrum commented:
You can release the unstable version as an alpha or beta version.

maltsev commented:
> So now basically the old version is up to date, including dependencies.

Awesome! Thanks a lot!

Yes, I think we can publish both `v0.1.2` and `v1.1.0`. If there are some bugs people can report them and downgrade to the previous stable version until it's fixed. I don't think it's worth releasing alpha or beta version considering not so many users use this lib.

cossssmin commented:
OK, I'll keep `0.1.x` as CJS and publish a patch with this PR's fix. Then I'll publish the hybrid as `1.1.0`.
